addons/mail/static/src/scss/composer.scss

```diff
@@ -280,180 +280,6 @@
     font-size: 100%;
 }
 
-// Attachment Icons (common for chat thread and chat composer)
-// ------------------------------------------------------------------
-$o-mail-attachment-image-size: 38px;
-$o-mail-attachment-margin: 5px;
-
-.o_attachments_list, .o_attachments_previews {
-    margin: 0 (-$o-mail-attachment-margin) 0;
-    display: flex;
-    flex-flow: row wrap;
-}
```

Comment:
those rules should stay in mail, i think (they apply on mail thread and chatter)

Reply:
Moved